python/pyspark/ml/classification.py

```diff
@@ -500,16 +500,12 @@ def featureImportances(self):
         """
         Estimate of the importance of each feature.
 
-        This generalizes the idea of "Gini" importance to other losses,
-        following the explanation of Gini importance from "Random Forests" documentation
-        by Leo Breiman and Adele Cutler, and following the implementation from scikit-learn.
+        Each feature's importance is the average of its importance across all trees in the ensemble
```

Comment:
Ah sorry I missed that.
On Wed, 30 Mar 2016 at 17:32, Seth Hendrickson <notifications@github.com>
wrote:

> In python/pyspark/ml/classification.py
> <https://github.com/apache/spark/pull/12056#discussion_r57909234>:
>
> > @@ -500,16 +500,12 @@ def featureImportances(self):
> >          """
> >          Estimate of the importance of each feature.
> >
> > -        This generalizes the idea of "Gini" importance to other losses,
> > -        following the explanation of Gini importance from "Random Forests" documentation
> > -        by Leo Breiman and Adele Cutler, and following the implementation from scikit-learn.
> > +        Each feature's importance is the average of its importance across all trees in the ensemble
>
> It was update in the PR I linked to above, so everything should be in sync.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12056/files/892d30123a192cd3796892c0f64a5cf2993e1f09#r57909234>
>